src/app/core/constants/routes.ts

```diff
@@ -32,6 +39,18 @@ export const AppRoutes: Routes = [
             path: 'users',
             component: UserManagementView,
           },
+          {
+            path: 'users/create',
+            component: CreateUserView,
+          },
+          {
+            path: 'users/:id',
+            component: EditUserView,
+          },
+          {
+            path: 'users/:id/update-pass',
+            component: UpdateUserPasswordView,
+          },
```

Comment:
in the future, please use more explicit param names like `userId` in order to improve readability when accessing in components.